CHANGELOG.md

```diff
@@ -4,13 +4,27 @@ TODO: Structopt and traits
 TODO: `Shell`, `YamlLoader`
 TODO: `cargo`, `std` features
 
+#### Minimum Required Rust
+* As of this release, `clap` requires `rustc 1.42.0` or greater.
```

Comment:
Just out of curiosity, what feature was Rust 1.40.0 missing?

Reply:
The `matches!()` macro

Reply:
Ah good point.

Reply:
We will probably move to at least 1.46 sometime soon. (See the issues tagged with it).